packages/launchpad/src/settings/SettingsCard.vue

```diff
@@ -1,9 +1,7 @@
 <template>
   <div class="p-4 min-w-650px mx-auto my-0">
-    <section class="border-1 border-gray-300 rounded">
-
-      <!-- The Settings Card Header -->
-      <header @click="toggleOpen()" data-testid="settings-card-header" class="bg-cool-gray-100 py-4 pl-6 pr-3 select-none cursor-pointer grid gap-4">
+    <Disclosure as="section" class="border-1 border-gray-300 rounded" v-slot="{ open }">
```

Comment:
I have a personal preference to use the hash syntax for scoped slots. I find it's more readable when dealing with named scoped slots and I want to pick one way of writing scoped slots.

That being said, so long as we pick a single way of doing it, I don't care.

```suggestion
    <Disclosure as="section" class="border-1 border-gray-300 rounded" #="{ open }">
```

Reply:
I wonder if we can make a PR to eslint-vue-plugin at some point to enforce things like this, one line per attr, `@` over `v-on`, etc. Agreed on consistency for sure.

Reply:
I'm fine with switching to `#=` for now as we already have it elsewhere. Oddly the [docs](https://v3.vuejs.org/guide/component-slots.html#named-slots-shorthand) suggest `#=` is invalid, though is doesn't actually trigger a warning for me. Both `#=` and `#default=` work fine. Maybe the docs are out of date, or I'm missing something.

@lmiller1990 I'm with you on some linting for stuff like this. I'm tweaking the format a little bit now in this PR. I'm happy with whatever linting we want to converge on though, that we can set & forget. I think there's some support out there for managing tailwind class lists that could be useful as well, which could be part of that convo.